src/pages/groups/Group.js

```diff
@@ -98,6 +98,7 @@ export const GroupDetails = ajaxCaller(class GroupDetails extends Component {
           adminLabel: 'admin',
           userLabel: 'member',
           title: 'Add user to Terra Group',
+          checkRegistration: true,
```

Comment:
This doesn't match any props in the component. Should this be `addUnregisteredUser`?